lib/guard/rspec/runner.rb

```diff
@@ -24,6 +25,7 @@ def initialize(options = {})
         @options = options
         @inspector = Inspectors::Factory.create(@options)
         @notifier = Notifier.new(@options)
+        @last_run_md5 = ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.